nautobot/extras/homepage.py

```diff
@@ -14,7 +14,7 @@ def get_job_results(request):
 
 def get_changelog(request):
     """Callback function to collect changelog for panel."""
-    return ObjectChange.objects.restrict(request.user, "view").select_related("user", "changed_object_type")[:15]
+    return ObjectChange.objects.restrict(request.user, "view").select_related("changed_object_type")[:15]
```

Comment:
I'm really curious how much of a performance improvement this change makes. The added index I can absolutely see making a huge difference but is one join really a significant issue?

Reply:
When @glennmatthews was pulling the `explain query` it was for some reason doing a sequential scan of the database.